app/helpers/application_helper.rb

```diff
@@ -25,7 +25,7 @@ def block_code(code, language)
 
   end
   def markdown(description)
-    renderer = HTMLwithPygments.new(hard_wrap: true, filter_html: true)
```

Comment:
is this related to the gem pygments.rb???
i got no clue.

Reply:
Its for redcarpet (blog markdown gem), the change lets me as the author insert html into the blog page. its secure as we're only going to have us as authors so its cool.

Reply:
Okay... you sound so convincing. 👍